bay_platform_dependencies.install

```diff
@@ -18,3 +18,46 @@ function bay_platform_dependencies_install() {
 function bay_platform_dependencies_uninstall() {
   \Drupal::messenger()->addStatus(__FUNCTION__);
 }
+
+/**
+ * Update section purger plugin to sectionbundled
+ */
+function bay_platform_dependencies_update_8001() {
```

Comment:
@FleetAdmiralButter FYI I updated the hook number. Can you please review this PR again.